docs/user-guide/node-selection/pod-with-pod-affinity.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: with-newlabels
+  annotations:
+    scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
There is syntactic validation; see elsewhere in this PR (e.g. validation.go)
Knowing whether an annotation could ever be met is probably not equivalent to the halting problem but may be close. Feel free to file an issue but I think it's outside the scope of a first version of this feature.